CMakeLists.txt

```diff
@@ -211,6 +211,7 @@ cprover_default_properties(
 
 option(WITH_JBMC "Build the JBMC Java front-end" ON)
 if(WITH_JBMC)
+    find_program(MAVEN_PROGRAM mvn)
```

Comment:
This change seems to be completely supplanted by the second commit? Maybe it'd make more sense to squash those.

Reply:
Done!